pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 100
-target-version = ['py36', 'py37', 'py38', 'py39']
\ No newline at end of file
+target-version = ['py36', 'py37', 'py38', 'py39', 'py310']
```

Comment:
Do we drop support for 3.6? Or `py36` is still required for some purposes?

Reply:
It is 'black' settings for supported code. Terra kept it but I don't think it makes sense if python 3.6 is not supported in main anymore. I removed it but need to be careful to put it back in stable once this gets backported, since stable still supports 3.6